### PR TITLE
New post - Mirror Conf 2018

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: pages/posts/2018/10-october/142-mirror-conf.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+This year the conference will take place once again in the beautiful city of Braga, Portugal’s third biggest city, with an incredible history and a fantastic University.
+
+Our five-day adventure will start with three days of workshops hosted by [Startup Braga](https://www.startupbraga.com/index.php?slang=en_US) at [GNRation](http://www.gnration.pt/en) ([map](https://goo.gl/maps/6AKGBBn5Wfu)).
+
+The conference itself will be hosted at [Forum Braga](https://www.forumbraga.com/) ([map](https://goo.gl/maps/Evs2LV3pWh82)), a brilliant renovated venue located in the heart of the city, designed for people and built to be modern, interactive, and environmentally sustainable. With one of the largest auditorium’s of the country, this will be the home of the biggest Mirror Conf yet.
+
+## Line-up
+
+Our line-up this year is as good as it gets, filled with some of the biggest names of the industry, including repeated presences in our program and newcomers alike. And if you want more, we invite you to come in early and extend your experience at Mirror Conf 2018 by attending a workshop by some of our speakers.
```
Comment (pfac) [inline pages/posts/2018/10-october/142-mirror-conf.md]: That last part is not sounding right. I suggest changing to `by one of our speakers`.

File: pages/posts/2018/10-october/142-mirror-conf.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+1. Amber Case - Designing Calm Technology (October 15th)
+2. Vitaly Friedman - Smart Responsive Interface Design Patterns (October 15th)
+3. Mike Monteiro - Design is a Job (October 16th)
+4. Stephanie Troeth - Jobs-to-Be-Done for Better Product Strategy (October 16th)
+5. Lea Verou - CSS in 2018 (October 17th)
+6. Jared Spool - Designing for Delight (October 17th)
+
+### Conference Line-up (October 18-19th)
+
+Mike Monteiro | Jared Spool | Stephanie Troeth | Vitaly Friedman | Amber Case | Lea Verou | Aras Bilgen | Arne Kittler | Jackie Balzer | Jessica Jordan | Vivianne Castillo | Sabine van der Eijk & Alina Leuca
```
Comment (pfac) [inline pages/posts/2018/10-october/142-mirror-conf.md:50]: Why not make this a list?

**Review by pfac — APPROVED** [review]
Just 2 tweaks, feel free to proceed if you disagree.

File: pages/posts/2018/10-october/142-mirror-conf.md (hunk 2 above)
Comment (naps62) [reply pages/posts/2018/10-october/142-mirror-conf.md:50]: this is what I was given. I'm not reviewing content at this point. take that up to management if you want :policeman:

File: pages/posts/2018/10-october/142-mirror-conf.md (hunk 1 above)
Comment (naps62) [reply pages/posts/2018/10-october/142-mirror-conf.md]: yep. forgot that change after our last discussion

**Pull request merged**